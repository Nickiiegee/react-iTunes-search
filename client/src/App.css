body {
  background: #eeeeee;
  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
}

.App {
  text-align: center;
}

.App-header {
  background-color: rgb(1, 172, 240);
  display: flex;
  align-items: center;
  justify-content: center;
  color: white;
  padding-bottom: 10px;
  padding-top: 10px;
  box-shadow: 1px 1px 1px 1px gray;
  position: fixed;
  width: 100%;
}

.search-bar {
  font-size: 15px;
  padding: 13px;
  border-radius: 2px;
  border: 1px solid gray;
  position: relative;
  width: 30vw;
}

.search-bar:focus {
  background-color: rgba(225, 225, 225, 0.979);
  box-shadow: 3px 3px 3px 3px #111;
}

.search-icon {
  position: relative;
  padding-right: 0.7rem;
  width: 40px;
  height: 40px;
}

.album {
  background-color: slategrey;
  margin: 20px;
  width: 20%;
  display: inline-block;
}

.album h1, h3, p {
  text-align: left;
  padding-left: 1rem;
  padding-right: 1rem;
}

.album h3 {
  font-style: italic;
  color: #fff;
}

.album img {
  width: 200px;
  height: 200px;
  padding-top: 1.5rem;
}

.zoom {
  transition: transform .2s;
  /* Animation */
  margin: auto;
  align-items: stretch;
  border-radius: 20px;
  padding: 1em;
}

.zoom:hover {
  transform: scale(1.2);
  /* (150% zoom) */
}

.dropdown {
  width: 10%;
  color: black;
}

.fav {
  margin-left: 200px;
  margin-bottom: 1rem;
  cursor: pointer;
}

.submit {
  background: none;
  border: 1px solid black;
  border-radius: 5px;
  box-shadow: 2px 2px 4px gray;
}

.submit:hover {
  color: #eeeeee;
  cursor: pointer;
}

#favouriteButton {
  margin-right: 25px;
}

.dropdown:hover {
  cursor: pointer;
}

.result {
  padding-top: 70px;
}