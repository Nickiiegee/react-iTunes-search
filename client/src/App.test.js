import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer';

test('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('test for component rendered', () => {
  const renderedComponent = renderer.create(<App />).toJSON()
  expect(renderedComponent).toMatchSnapshot()
});