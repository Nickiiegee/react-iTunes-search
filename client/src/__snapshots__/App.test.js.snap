// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for component rendered 1`] = `
<div
  className="App"
>
  <div>
    <form
      className="App-header"
    >
      <div
        className="favourites"
      >
        <button
          aria-expanded="false"
          aria-haspopup="true"
          className="btn btn-secondary dropdown-toggle"
          data-toggle="dropdown"
          id="favouriteButton"
          type="button"
        >
          Favourites:
        </button>
        <p>
          No Favourites Yet
        </p>
      </div>
      <svg
        className="search-icon"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
        />
      </svg>
      <input
        className="search-bar"
        name="query"
        onChange={[Function]}
        type="text"
      />
      <div
        className="dropdown"
      >
        <button
          aria-expanded="false"
          aria-haspopup="true"
          className="btn btn-secondary dropdown-toggle"
          data-toggle="dropdown"
          id="dropdownMenuButton"
          type="button"
        >
          Type
        </button>
        <div
          aria-labelledby="dropdownMenuButton"
          className="dropdown-menu"
        >
          <a
            className="dropdown-item"
            href="#1"
            onClick={[Function]}
          >
            music
          </a>
          <a
            className="dropdown-item"
            href="#2"
            onClick={[Function]}
          >
            movie
          </a>
          <a
            className="dropdown-item"
            href="#3"
            onClick={[Function]}
          >
            podcast
          </a>
          <a
            className="dropdown-item"
            href="#4"
            onClick={[Function]}
          >
            shortFilm
          </a>
          <a
            className="dropdown-item"
            href="#5"
            onClick={[Function]}
          >
            ebook
          </a>
        </div>
      </div>
      <input
        className="submit"
        onClick={[Function]}
        type="button"
        value="🔍 Search"
      />
    </form>
    <div
      className="result"
    />
  </div>
</div>
`;
